packages/pages/src/scaffold/template/generate.ts

```diff
@@ -0,0 +1,117 @@
+import prompts, { PromptObject } from "prompts";
+import { ProjectStructure } from "../../common/src/project/structure.js";
+import path from "node:path";
+import fs from "node:fs";
+
+/* eslint-disable @typescript-eslint/no-unused-vars */
+// TODO: Remove after using adding generation code for templates
+
+export const generateTemplate = async (
+  projectStructure: ProjectStructure
+): Promise<void> => {
+  const questions: PromptObject[] = [
+    {
+      type: "text",
+      name: "templateName",
+      message: "What would you like to name your Template?",
+      validate: (templateName) =>
+        validateTemplateName(templateName, projectStructure) ||
+        "Please ensure the name provided isn't already used and is valid.",
+    },
+    {
+      type: "confirm",
+      name: "isVisualEditor",
+      message: "Is this a Visual Editor template?",
+      initial: true,
+    },
+    {
+      type: (prev) => (prev ? null : "toggle"),
+      name: "isDynamic",
+      message: "Is this a static or dynamic template?",
+      initial: true,
+      active: "Dynamic",
+      inactive: "Static",
+    },
+    {
+      type: (prev) => (prev ? "select" : null),
+      name: "entityScope",
+      message:
+        "How would you like you to define the entity scope for your template?",
+      choices: [
+        { title: "Entity Type", value: "entityTypes" },
+        { title: "Static", value: "static" },
```

Comment:
Static isn't a scope option. Aaron wrote "Static Filter" but he fixed after I commented. It should be "Saved Filter".